testing/iozone/iozone.patch

```diff
@@ -0,0 +1,54 @@
+84a85
+> #include <sys/types.h>
```

Comment:
let's generate the patch by "git format HEAD~"?

Reply:
Done!